app/src/main/kotlin/com/hippo/ehviewer/ui/main/EhAsyncThumb.kt

```diff
@@ -19,7 +19,7 @@ import com.hippo.ehviewer.ui.tools.shouldCrop
 @Composable
 fun requestOf(model: GalleryInfo): ImageRequest {
     val context = LocalContext.current
-    return remember { context.imageRequest(model) }
+    return remember(context, model) { context.imageRequest(model) }
```

Comment:
Why do we need the `context` as key?
IIRC `context` is always the root `ComposeView`?

Reply:
Not necessarily needed, just for aesthetics.